airbyte-cdk/python/airbyte_cdk/sources/declarative/auth/__init__.py

```diff
@@ -3,7 +3,9 @@
 #
 
 from airbyte_cdk.sources.declarative.auth.oauth import DeclarativeOauth2Authenticator
+from airbyte_cdk.sources.declarative.auth.jwt import JwtAuthenticator
 
 __all__ = [
     "DeclarativeOauth2Authenticator",
```

Comment:
I wonder why SelectiveAuthenticator isn't here.

Reply:
I don't have a good answer for you. Oauth is re-exported but all other authenticators are accessed directly from their files.

Reply:
Probably not intentional. It shouldn't matter as these classes should only be used through the YAML interface